controllers/reconcilers/grafana/deployment_reconciler.go

```diff
@@ -20,10 +20,6 @@ import (
 )
 
 const (
-	InitMemoryRequest                       = "128Mi"
-	InitCpuRequest                          = "250m"
-	InitMemoryLimit                         = "512Mi"
-	InitCpuLimit                            = "1000m"
```

Comment:
These values aren't used so I removed them.

Reply:
:+1: they must have been ported over from v4, no need to keep them here